scripts/tests/java/commissioning_test.py

```diff
@@ -93,6 +93,26 @@ def RunTest(self):
             code = self.TestCmdOnnetworkLong(self.nodeid, self.setup_pin_code, self.discriminator, self.timeout)
             if code != 0:
                 raise Exception(f"Testing pairing onnetwork-long failed with error {code}")
+        elif self.command_name == 'onnetwork-long-im-invoke':
+            logging.info("Testing pairing onnetwork-long-im-invoke")
+            code = self.TestCmdOnnetworkLongImInvoke(self.nodeid, self.setup_pin_code, self.discriminator, self.timeout)
+            if code != 0:
+                raise Exception(f"Testing pairing onnetwork-long-im-invoke failed with error {code}")
+        elif self.command_name == 'onnetwork-long-im-write':
+            logging.info("Testing pairing onnetwork-long-im-write")
+            code = self.TestCmdOnnetworkLongImWrite(self.nodeid, self.setup_pin_code, self.discriminator, self.timeout)
+            if code != 0:
+                raise Exception(f"Testing pairing onnetwork-long-im-write failed with error {code}")
+        elif self.command_name == 'onnetwork-long-im-read':
+            logging.info("Testing pairing onnetwork-long-im-read")
+            code = self.TestCmdOnnetworkLongImRead(self.nodeid, self.setup_pin_code, self.discriminator, self.timeout)
+            if code != 0:
+                raise Exception(f"Testing pairing onnetwork-long-im-read failed with error {code}")
+        elif self.command_name == 'onnetwork-long-im-subscribe':
+            logging.info("Testing pairing onnetwork-long-im-subscribe")
+            code = self.TestCmdOnnetworkLongImSubscribe(self.nodeid, self.setup_pin_code, self.discriminator, self.timeout)
+            if code != 0:
+                raise Exception(f"Testing pairing onnetwork-long-im-subscribe failed with error {code}")
```

Comment:
Why those commands are triggers from im cluster in the test.yaml, but implemented within commissioning_test?

`--tool-cluster "im" \`

It looks you have already added 'scripts/tests/java/im_test.py' for those tests.

Reply:
need to be removed. thanks

Reply:
remove, thanks